sacred/exception.py

```diff
@@ -0,0 +1,223 @@
+import inspect
+import sys
+from collections import defaultdict
+
+from sacred.config.config_sources import ConfigSource
+
+if colored_exception_output:
```

Comment:
This variable is not defined anywhere. You import it from utils later, but also there it is not defined. Should this maybe be in settings? Or did you plan on auto-detecting if the console supports color?